CHANGELOG.md

```diff
@@ -8,6 +8,29 @@ Changes should start with one of the following tags:
 - `[changed]` for changes to the existing API
 - `[removed]` for keys that have been removed from the schema
 
+## v15 (unreleased)
+
+Root level:
+
+- [added] The `linked_spaces` field was added ([#75])
```

Comment:
Would you add links to all PRs which are about `linked_spaces`? https://github.com/SpaceApi/schema/pull/92 for example?

Reply:
Ah, good catch. If there's an issue and a PR, I'd only reference one of them, but if there are multiple PRs, then all of them should be listed of course.